receiver/apachedruidreceiver/metadata.yaml

```diff
@@ -0,0 +1,2541 @@
+type: apachedruid
+
+status:
+  class: receiver
+  stability:
+    development: [metrics, logs]
+  distributions: [contrib, observiq, sumo]
+  codeowners:
```

Comment:
There should be an added entry in the [CODEOWNERS](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/.github/CODEOWNERS) file for this component.

Reply:
Updated